app/helpers/access_lint_helper.rb

```diff
@@ -0,0 +1,19 @@
+module AccessLintHelper
+  def include_access_lint
+    if Rails.env.production?
+      js_file = "//accesslint.com/1.0/accesslint.js"
```

Comment:
Should we serve this from the same place we're reporting errors to? Or a CDN?

Maybe that's a future change, but moving it to the service app would let us do clever user-specific dynamic things in the JavaScript, e.g. some of the sampling stuff we were talking about.